apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/model/CreateTransactionRequest.java

```diff
@@ -41,8 +42,8 @@
 
     private long feeATM;
     private long amountATM;
-    private Integer ecBlockHeight;
-    private Long ecBlockId;
+    private int ecBlockHeight;
+    private long ecBlockId;
```

Comment:
It would be good to update similar fields in new REST API DTOs (if that is not updated yet). It seems they might be exist, but it's better to check twice. If that DTO exists in new code, pls update relevant code accordingly.

Reply:
It refers to transaction request not to dtos, so that it does not make a sense.